### PR TITLE
[Ide] Clear BuildOutput when clean operations start

### PR DESCRIPTION
**Pull request by rodrmoya**
Fixes https://devdiv.visualstudio.com/DevDiv/_workitems/edit/641608

### CODE REVIEW

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads/ErrorListPad.cs [hunk 1]
```diff
@@ -249,6 +249,7 @@ protected override void Initialize (IPadWindow window)
 			UpdatePadIcon ();
 
 			IdeApp.ProjectOperations.StartBuild += OnBuildStarted;
+			IdeApp.ProjectOperations.StartClean += OnBuildStarted;
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads/ErrorListPad.cs]: Event is not removed:
https://github.com/mono/monodevelop/blob/25b2bd179ebff27af07119638172225430b4f3d7/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads/ErrorListPad.cs#L346

Comment (rodrmoya) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads/ErrorListPad.cs]: ugh right, fixing

Comment (rodrmoya) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads/ErrorListPad.cs]: fixed

**Comment by Therzok** [comment]
@monojenkins backport release-7.6

**Comment by Therzok** [comment]
@monojenkins backport release-7.7

**Comment by parasAmbhore** [comment]
Issue was not reproducible using shared steps.
Hence marking it as QA approved.

=== Visual Studio Enterprise 2017 for Mac (Preview) ===

Version 7.8 Preview (7.8 build 43)
Mono 5.10.1.57 (2017-12/ea8a24b1bbf) (64-bit)
Xamarin.Mac 4.4.1.178 (master / eeaeb7e6)

=== Xamarin.Mac ===
Version: 4.4.1.193 (Visual Studio Enterprise)
=== Xamarin.iOS ===
Version: 11.14.0.11 (Visual Studio Enterprise)
=== Xamarin.Android ===
Version: 8.3.3.2 (Visual Studio Enterprise)

Build Taken from :
https://dl.internalx.com/artifacts/devdiv-1927476/VisualStudioForMac-Preview-7.8.0.43.dmg

Screen Cast: https://www.screencast.com/t/Fc09ckGMvkfz

Gist Link: https://gist.github.com/mohakbarokar/bc0f2b71ce9196cfd16e0a646ef972ed

**Pull request merged**